build.gradle.kts

```diff
@@ -113,6 +113,7 @@ tasks.jar {
         attributes(
             "Bundle-License" to "https://apache.org/licenses/LICENSE-2.0",
             "Bundle-Description" to "TestNG is a testing framework.",
+            "Bundle-Version" to This.version,
```

Comment:
Just add a comment that explains the version in the manifest is important for the IntelliJ plugin

Reply:
I didnt understand this. Where do you want me to add the comment ?

Reply:
Just after `This.version` in order to remember why we need it